### PR TITLE
fixing the damage formula

### PR DESCRIPTION
**Pull request by EmilysPaimon**
tl;dr i tried

this is scuffed but not as scuffed as the formulas being cropped off or just using screenshots imo. hopefully the white background on the embedded images isn't noticeable on the tcl, it does look weird when editing/previewing the page in dark mode though. but if the images had transparent backgrounds they'd just be really hard to see in dark mode. feel free to fix this fix if anyone can find a better way to solve the problem

commented out the details at the beginning of the page but here's a link to all the formulas just in case: https://docs.google.com/document/d/1ljOorhWjHAqj7lmrPxI0c0mojBQXb13XPmRln25S8Jo/edit?usp=sharing

screenshots of the current formulas on the tcl that idk what to do with because they're all different sizes: https://imgur.com/a/P9hqmzr

### CODE REVIEW

**Comment by Artesians** [comment]
WICKED - we'llsee what it looks like on live 🔍
